### PR TITLE
tiledb: Fix dependencies, Use Ninja.

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

**Comment by Biswa96** [comment]
@jeroen Your opinion please.

**Comment by jeroen** [comment]
I had to use `export MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX="` to fix the install dir, is that no longer needed when using ninja?

**Comment by Biswa96** [comment]
Seems not to. Maybe it's make vs ninja thing.

**Comment by Biswa96** [comment]
The build has complete. Now you can check the difference with previous commit.

**Comment by jeroen** [comment]
Why does the static library increase from 27 to 45mb? Does ninja include debug symbols?

**Comment by Biswa96** [comment]
No, debug is disabled. See the negation `options` list. AFAIK, the static library is not stripped.

File: mingw-w64-tiledb/PKGBUILD [hunk 1]
```diff
@@ -12,69 +14,87 @@ depends=("${MINGW_PACKAGE_PREFIX}-aws-sdk-cpp"
...
-  export MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX="
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
   ${MINGW_PREFIX}/bin/cmake.exe \
-    -G"MSYS Makefiles" \
+    -Wno-dev \
+    -G "Ninja" \
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
-    -DCMAKE_BUILD_TYPE=Release \
+    "${extra_config[@]}" \
     -DTILEDB_STATIC=ON \
```
Comment (lazka) [inline mingw-w64-tiledb/PKGBUILD:78]: TILEDB_STATIC is needed for the shared build as well?

Comment (jeroen) [reply mingw-w64-tiledb/PKGBUILD:78]: Ah probably not... (I'm only using static builds myself so I didn't notice).

**Pull request merged**